ts/screens/preferences/OldServiceDetailsScreen.tsx

```diff
@@ -440,7 +441,9 @@ class OldServiceDetailsScreen extends React.Component<Props, State> {
               renderInformationRow(
                 I18n.t("services.contactAddress"),
                 address,
-                () => Clipboard.setString(address)
+                () => {
+                  openMaps(address);
+                }
```

Comment:
@Alex193a Could you fix also the typo in line 453?
from **EMail** to **Email**

Reply:
Fixed! [5a8d1ef](https://github.com/teamdigitale/io-app/pull/1174/commits/5a8d1efbc2b217aaf4260d2926d85619c25b4aed)